packages/core/src/utils/spanUtils.ts

```diff
@@ -127,6 +131,7 @@ export function spanToJSON(span: Span): Partial<SpanJSON> {
         op: attributes[SEMANTIC_ATTRIBUTE_SENTRY_OP],
         origin: attributes[SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN] as SpanOrigin | undefined,
         _metrics_summary: getMetricSummaryJsonForSpan(span),
+        exclusive_time: attributes[SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME] as number | undefined,
```

Comment:
So far, the attribute we're looking at is not called `sentry.exclusive_time`. @cleptric suggested to rename it more in line with how OTel timestamps are named so this is coming https://github.com/getsentry/relay/pull/3521.

I suggest we rename it here as well.

Reply:
`exclusive_time_nano` is fine, the Sentry prefix is meant for Otel. In Relay, we check for `key if key.contains("exclusive_time_nano") ...` so both should work 🙂